aerleon/lib/naming.py

```diff
@@ -357,7 +360,7 @@ def _GetParents(self, query, query_group):
                 recursive_parents.append(bp)
         return recursive_parents
 
-    def GetNetChildren(self, query):
+    def GetNetChildren(self, query: str) -> List[Union[str, Any]]:
```

Comment:
Can the "Any" be made more specific? Looking at the code, the return list members all come from `child = item.split('#')[0].strip()` which feels pretty specific.

Reply:
You are right, strangely the tests never return anything but a string.